react-example/src/Screens/Login/Login.jsx

```diff
@@ -0,0 +1,94 @@
+// External modules
+import React, { useEffect, useState } from 'react'
```

Comment:
We don't import `React` functions, we [just import react](https://github.com/DeviasRioIV/frontend-style-guide/blob/develop/react-web.md#-external-modules) and call it's functions in the form `React.useEffect` for example.

Please fix this in the rest of files as well.